react/src/components/FolderCreateModal.tsx

```diff
@@ -135,6 +135,9 @@ const FolderCreateModal: React.FC<FolderCreateModalProps> = ({
             document.dispatchEvent(
               new CustomEvent('backend-ai-folder-list-changed'),
             );
```

Comment:
I think this dispatch part can be removed as `StorageStatusPanel` uses an independent fetchKey :)